Makefile

```
@@ -145,12 +145,11 @@ CI_GOLANG_DOCKER_MAKE := $(DOCKER) run \
         -e GOCACHE=/src/.go/cache \
         -e CGO_ENABLED=$(CGO_ENABLED) \
 		-e GOARCH=$(GOARCH) \
-        golang:$(GOVERSION) \
-		make
+        golang:$(GOVERSION)
 
 .PHONY: ci-go-%
 ci-go-%:
-	$(CI_GOLANG_DOCKER_MAKE) "$*"
+	$(CI_GOLANG_DOCKER_MAKE) /bin/bash -c "git config --global --add safe.directory /src && make $*"
```

```suggestion
	/bin/bash -c "git config --global --add safe.directory /src && $(CI_GOLANG_DOCKER_MAKE) $*"
```
This way, we avoid accidentally invoking the *user's* `make` (or causing an error if it's not available, like on some Windows systems).

---

I'm not sure I understand. The `/src` directory in mounted inside the container and hence we run the command within the container. Maybe I'm missing something. [This](https://github.com/open-policy-agent/opa/blob/main/Makefile#L269) is what I referred to.